paymentsheet/src/main/java/com/stripe/android/link/account/DefaultLinkAuth.kt

```diff
@@ -130,13 +156,15 @@ internal class DefaultLinkAuth @Inject constructor(
     }
 
     private val Throwable.isAttestationError: Boolean
-        get() = when (this) {
-            // Stripe backend could not verify the intregrity of the request
-            is APIException -> stripeError?.code == "link_failed_to_attest_request"
-            // Interaction with Integrity API to generate tokens resulted in a failure
-            is AttestationError -> true
-            else -> false
-        }
+        get() = isIntegrityManagerError || isBackendAttestationError
+
+    // Interaction with Integrity API to generate tokens resulted in a failure
+    private val Throwable.isIntegrityManagerError: Boolean
+        get() = this is AttestationError
+
+    // Interaction with Integrity API to generate tokens resulted in a failure
```

Comment:
Should this be this?

```suggestion
    // Stripe backend could not verify the intregrity of the request
```